scylla-server-rust/src/controllers/run_controller.rs

```diff
@@ -25,7 +25,7 @@ pub async fn get_run_by_id(
     let run_data = run_service::get_run_by_id(&db, run_id).await?;
 
     if run_data.is_none() {
-        return Err(ScyllaError::NotFound);
+        return Err(ScyllaError::EmptyResult);
```

Comment:
why the name change?

Reply:
because not found is an error I used for when a query cannot find something.  On other other hand this error should not be that, instead it should represent when runs are found but the result is empty.